modules/sql-engine/src/integrationTest/sql/subquery/table/union_all_projection_pushdown.test

```diff
@@ -0,0 +1,91 @@
+# name: union_all_projection_pushdown.test
+# description: Test projection pushdown in UNION ALL
+# group: [setops]
```

Comment:
please add a reference to the E071-01 SQL feature

Reply:
done